e2e-tests/portal/cypress/e2e/journeys/payment-report-officer/utilisation-report-upload/confirm-and-send.spec.js

```diff
@@ -23,13 +23,15 @@ context('Confirm and send', () => {
 
       confirmAndSend.mainHeading().should('not.exist');
       confirmAndSend.currentUrl().should('contain', '/utilisation-report-upload');
+      confirmAndSend.currentUrl().should('not.contain', '/confirm-and-send');
```

Comment:
I've added this because our tests which want to navigate to the `Upload Report page` will pass even if we navigate to `/utilisation-report-upload/confirm-and-send` as that url does contain `/utilisation-report-upload`

Reply:
I imagine we would need to use a regex to do that, as our url can sometimes end with our `?_csrf=` query